src/test/FuzzerImpl.cpp

```diff
@@ -19,6 +19,8 @@
 #include "util/Logging.h"
 #include "util/Math.h"
 #include "util/XDRCereal.h"
+#include "util/optional.h"
```

Comment:
Nitpick: I don't think this is used.

Reply:
> Nitpick: I don't think this is used.

Good catch.  (I'd used it in a previous iteration of the code, then changed it not to use `optional`.)  Removed.